Data/models/DataController.swift

```diff
@@ -206,6 +206,30 @@ public class DataController: NSObject {
         storeDescription.setOption(completeProtection, forKey: NSPersistentStoreFileProtectionKey)
         
         container.persistentStoreDescriptions = [storeDescription]
+
+        // Set the security policy to exclude the database from backup
+        do {
+            var store = store
+            var resourceValues = URLResourceValues()
+            resourceValues.isExcludedFromBackup = true
+            try store.setResourceValues(resourceValues)
+        } catch {
+            log.error("Could not set security policy on url for: \(store)")
+        }
+        
+        // Set the security policy to exclude the ApplicationSupportDirectory from backup
+        let directory = FileManager.SearchPathDirectory.applicationSupportDirectory
+        let urls = FileManager.default.urls(for: directory, in: .userDomainMask)
+        
+        for var url in urls {
```

Comment:
Do we need to do that for each separate file? Could we just set backup exclusion at folder level?
Not sure if this can be done to `application support` dir

Reply:
I wasn't sure if it works on the entire folder either which is why I left it on all the files. I can try and see if it works on the folder.

Reply:
Would be great if you could check it